src/app/settings/admin/interface/template.html

```diff
@@ -82,7 +82,16 @@
         </div>
         <mat-checkbox [(ngModel)]="settings.enableDashboard"
                       (change)="onSettingsChange()"
-                      id="km-enable-kubernetes-dashboard-setting"></mat-checkbox>
+                      [disabled]="!isKonnectivityServiceEnabled || !isOpenIDAuthPluginEnabled"
+                      id="km-enable-kubernetes-dashboard-setting">
+          <mat-hint *ngIf="!isKonnectivityServiceEnabled || !isOpenIDAuthPluginEnabled">
+            The Kubernetes Dashboard requires the
+            <ng-container *ngIf="!isKonnectivityServiceEnabled">konnectivityService </ng-container>
+            <ng-container *ngIf="!isKonnectivityServiceEnabled && !isOpenIDAuthPluginEnabled">and </ng-container>
+            <ng-container *ngIf="!isOpenIDAuthPluginEnabled">openIDAuthPlugin</ng-container>
```

Comment:
Should we wrap `konnectivityService` and `openIDAuthPlugin` inside `code` block?